integration_tests/integration_tests.feature

```diff
@@ -79,9 +79,9 @@ Feature: Run integration tests
      Then we test the export functionality of default_2a
 
   Scenario: Simulation returns same results for different market counts
-    Given we have a scenario named default_2a
+    Given we have a scenario named default_2b
```

Comment:
Sorry, what does this change solve?

Reply:
We tried different profiles when testing this, it should be possible by now to revert to the old profile. I agree to the comment btw.